pkg/cmd/root.go

```diff
@@ -26,8 +26,7 @@ func Root(p cli.Params) *cobra.Command {
 	var cmd = &cobra.Command{
 		Use:   "tkn",
 		Short: "CLI for tekton pipelines",
-		Long: `
-	`,
+		Long:  ``,
```

Comment:
nit for future: we need to add descriptions here :angel:

Reply:
In which PR?

Reply:
later on :wink:

Reply:
:bowing_man: